src/items/FilePdf.tsx

```diff
@@ -41,11 +43,13 @@ const FilePdf: FC<FilePdfProps> = ({
     }
   };
 
+  const urlWithPdfViewer = `${PDF_VIEWER_LINK}${encodeURIComponent(url)}`;
+
   return (
     <embed
```

Comment:
Is there a reason why we use an embed and not an iframe? If we provide an external domain, we can also add additional security such as sandboxing.